runtests

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e -x
+
+pulp test
```

Comment:
Could we set the path variable so that it'll use the pulp & purs from the project? This script probably wouldn't work on my machine because my purs is out of date

Reply:
I don't think it makes sense to override the path here too... Have you tried `npm run test` or `direnv`?